features/locatie-zoeken.feature

```diff
@@ -0,0 +1,98 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik een pand kunnen zoeken op locatiecoördinaten
+    Zodat ik gegevens over het pand kan raadplegen door op het pand in een kaart te klikken 
+
+
+Rule: De standaardwaarde voor het coördinatenstelsel (CRS) voor de locatieparameter is Rijksdriehoekscoördinaten New Nederlands
+
+    Scenario: zoek op locatie zonder het coördinatenstelsel aan te geven
+        Gegeven het pand met identificatie "0826100000036343" heeft de volgende geometrie polygoon coördinaten
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?locatie=118320,404848" wordt gezocht zonder requestheader "Content-Crs"
+        Dan bevat het resultaat het pand met identificatie "0826100000036343"
+
+    Scenario: zoek op locatie met het coördinatenstelsel dat je gebruikt in de locatieparameter
+        Gegeven het pand met identificatie "0826100000036343" heeft de volgende geometrie polygoon coördinaten
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?locatie=118320,404848" wordt gezocht met requestheader "Content-Crs" met waarde "epsg:28992"
+        Dan bevat het resultaat het pand met identificatie "0826100000036343"
+
+Rule: Pand wordt gevonden wanneer de coördinaat van de locatieparameter binnen de pandgeometrie valt
+
+    Scenario: coördinaat locatie valt binnen geometrievlak van gezochte pand
+        Gegeven het pand met identificatie "0826100000036343" heeft de volgende geometrie polygoon coördinaten
+        | geometriecoördinaten                                                                                                        |
+        | [ [118315.606,404844.967],[118324.183,404843.509],[118325.179,404849.365],[118316.604,404850.835],[118315.606,404844.967] ] |
+        Als met "GET" "/panden?locatie=118320,404848" wordt gezocht
+        Dan bevat het resultaat het pand met identificatie "0826100000036343"
+
+    Scenario: coördinaat locatie valt buiten geometrievlak van gezochte pand
```

Comment:
Deze scenario geeft wel het betreffende pand terug. De locatie is hetzelfde als de vorige scenario

Reply:
ik maak ervan /panden?locatie=118314,404852